r-base/README.md

```diff
@@ -0,0 +1,92 @@
+# Clear Linux* OS `r-base` container image
+
+<!-- Required -->
+## What is this image?
+
+`clearlinux/r-base` is a Docker image with `r-base` running on top of the
+[official clearlinux base image](https://hub.docker.com/_/clearlinux). 
+
+<!-- application introduction -->
+> [R](https://www.r-project.org/) is a language and environment for statistical computing and graphics. It is a GNU project which is similar to the S language and environment which was developed at Bell Laboratories (formerly AT&T, now Lucent Technologies) by John Chambers and colleagues. R can be considered as a different implementation of S. There are some important differences, but much code written for S runs unaltered under R.
+
+For other Clear Linux* OS
+based container images, see: https://hub.docker.com/u/clearlinux
+
+## Why use a clearlinux based image?
+
+<!-- CL introduction -->
+> [Clear Linux* OS](https://clearlinux.org/) is an open source, rolling release
+> Linux distribution optimized for performance and security, from the Cloud to
+> the Edge, designed for customization, and manageability.
+
+Clear Linux* OS based container images use:
+* Optimized libraries that are compiled with latest compiler versions and
+  flags.
+* Software packages that follow upstream source closely and update frequently.
+* An aggressive security model and best practices for CVE patching.
+* A multi-staged build approach to keep a reduced container image size.
+* The same container syntax as the official images to make getting started
+  easy. 
+
+To learn more about Clear Linux* OS, visit: https://clearlinux.org.
+
+<!-- Required -->
+## Deployment:
+
+### Deploy with Docker
+The easiest way to get started with this image is by simply pulling it from
+Docker Hub. 
+
+*Note: This container uses the same syntax as the [official r-base image](https://hub.docker.com/_/r-base).
+
+
+1. Pull the image from Docker Hub: 
+    ```
+    docker pull clearlinux/r-base
+    ```
+
+2. Start a container using the examples below:
+
+    ```
+    docker run -d clearlinux/r-base
```

Comment:
What is this command for? "-d" means running in background.

Reply:
Yes, you are right. Can't use "-d" here, I have updated the patch again.
Thanks.